ui/components/component-library/tag/README.mdx

```diff
@@ -0,0 +1,19 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+
+import { Tag } from './tag';
+
+# Tag
+
+The `Tag` is a component used to display text within a container.
+
+<Canvas>
+  <Story id="ui-components-component-library-tag-tag-stories-js--default-story" />
+</Canvas>
+
+## Props
+
+<ArgsTable of={Tag} />
+
+### label
```

Comment:
Title Case for docs headings. Sorry I know this is contrary to passed conventions 

```suggestion
### Label
```